README.md

```diff
@@ -29,7 +30,8 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router has no impact on controller actions.  It merely directs a request to the proper controller.
+The controller decides what action to take.
 ```
```

Comment:
Not exactly, the router maps a request directly to a controller action. For example:

```ruby
get '/artists', to: 'artists#index'
```
Will map any request to the url `/artists` to the `index` action in the `Artists` controller